packages/website/src/components/playground.html

```diff
@@ -1,91 +1,482 @@
-<div class="w-full">
-  <style>
-      .CodeMirror {
-          font-size: 14px;
-      }
-      .editor_error {
-  text-decoration: underline dashed red;
-}
-  </style>
-  <h1 class="sr-only">Playground</h1>
-  <div class="flex flex-col mt-3 px-2 w-full ">
-    <div class="flex" id="language-tabs">
-      <div class="py-2 px-4 rounded-t cursor-pointer" data-language="html">
-        <h2>HTML</h2>
-      </div>
-      <div class="p-2 py-2 px-4 rounded-t cursor-pointer" data-language="javascript">
-        <h2>JavaScript </h2>
-      </div>
-    </div>
-    <div class="flex-1 min-h-[300px] border-[1px] border-slate rounded md:min-w-[400px] mb-2">
-      <textarea id="code-editor"></textarea>
-    </div>
-    <div class="flex flex-col md:flex-row flex-1 h-[400px] gap-2">
-      <div class="flex-1 min-h-[300px]">
-        <div>
-          <span class="inline-block p-2 rounded-t">.eslintrc.json</span>
-        </div>
-        <div class="border-[1px] border-slate rounded md:min-w-[400px] h-[400px]">
-          <textarea id="config-editor"></textarea>
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Code Playground</title>
+    <script src="https://cdn.tailwindcss.com"></script>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/codemirror/5.65.2/codemirror.min.css">
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/codemirror/5.65.2/codemirror.min.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/codemirror/5.65.2/mode/htmlmixed/htmlmixed.min.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/codemirror/5.65.2/mode/javascript/javascript.min.js"></script>
+    <!-- ESLint for real linting and auto-fix -->
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/eslint/8.56.0/eslint.min.js"></script>
```

Comment:
Since `layout.html` is used as a template via `posthtml`, the `html` & `head` & `body` tag is unnecessary. Also, `codemirror` is already specified in `package.json`, so there's no need to include it via CDN. Could you please remove these parts?